docs/manual/manual_redundant_throttle.rst

```diff
@@ -38,10 +43,17 @@ There are two types of configurations available to redundant throttle. The first
 source if the active source (the source whose throttles are being applied to module control) goes offline. The second set of parameters defines the priorities of each supported throttle source. You can find the specific parameters 
 at :ref:`throttle_source_manager`. You can access these parameters either using a :ref:`Vertiq API <getting_started_with_apis>` or through the :ref:`IQ Control Center <control_center_start_guide>`. 
 
-The time that a module waits (throttle source timeout) can be configured through the Control Center's tuning tab:
+The time that a module waits, its throttle switchover time, can be configured through the Control Center's tuning tab as ``Throttle Source Timeout``:

```

Comment:
avoid using the word timeout as much as possible...meaning everywhere besides direct interaction with the API or control center.